client/src/route/index.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from "react";
+import {
+    Router,
+    Route,
+    Link,
+    // Redirect,
+} from "react-router-dom";
+
+import history from "./history";
+import LoginPage from "../modules/auth";
+// import LessonList from "./modules/lesson/list/LessonList";
+// import LessonForm from "./modules/lesson/form/LessonForm";
+// import LessonDelete from "./modules/lesson/form/LessonDelete";
+
+import initialize from "../config/init";
+
+class AppRouter extends Component {
+    constructor(props) {
+        super(props);
+        initialize();
+    }
+
+    render() {
+        // todo research and use Refs instead of direct DOM manipulation
+        // todo refactor (id for getElementByID) >> (ref)
+        return (
+            <Router history={history}>
+                <div className={"router-root"}>
+                    <nav className={"router-nav-main"}>
```

Comment:
navigation is not suitable to be inside routes

Reply:
If i understood correctly Links/NavLinks must be outside of <Router ... /> tag ?

Reply:
Will consider the refactoring for better fit , during bootstrap research.

commit https://github.com/Levayv/ipma_v2/commit/699e165bf482b16f63cbe726aba48ba47cd124e7#diff-6bc026867844ef261a708695bac6f53fR22